assessment/libs/dates.php

```diff
@@ -0,0 +1,143 @@
+<?php
+// A library of date Functions.  Version 1.  December 17, 2020
+// Author:  Daniel Brown
+//          University of Colorado Boulder
+
+global $allowedmacros;
+array_push($allowedmacros,"dates_addDays","dates_addWeeks","dates_addMonths","dates_addYears","dates_EOMonth",
+"dates_BOMonth","dates_diffDays","dates_randdate","dates_dateFormat");
+
+// function addDays
+// adds a number of days to an existing date
+// uses $fmt as format of date inputs and outputs
+function dates_addDays($date, $days,$fmt = "F j, Y"){
+	$datetime = date_create_from_Format($fmt,$date);
```

Comment:
I think the function name is all lower case.  This applies everywhere this function is used.

Reply:
Just to make sure, are you saying you want functions to be all lowercase rather than mixed upper and lower, or that I have mixed upper and lower incorrectly somewhere with dates_addDays?

thanks,
Daniel

Reply:
apologies - I completely misread your comment.  I will change the function names to lower case.